sites/public/src/components/listing/ListingView.tsx

```diff
@@ -310,7 +310,12 @@ export const ListingView = (props: ListingProps) => {
   }
 
   let lotterySection
-  if (publicLottery && (!lotteryResults || (lotteryResults && !lotteryResults.url))) {
+
+  if (
+    publicLottery &&
+    (!lotteryResults || (lotteryResults && !lotteryResults.url)) &&
+    (listing.status === ListingsStatusEnum.active || !lotteryResults)
```

Comment:
issue: I was still able to reproduce the issue by doing an abnormal workflow.
1. Edit a listing to have a public lottery start date/time in the past
2. Close that listing and upload a lottery result
3. Re-open the listing. 
4. The "Invalid date" section is still being shown.

I think your changes are good here, but I believe we should also delete the entire section starting on line 326. It's using the wrong field to show the timestamp and also the above scenario is the only time it could be shown. So I'm good with you deleting the entire second EventSection